src/controller/timeline-controller.ts

```diff
@@ -196,12 +206,13 @@ class TimelineController extends EventHandler {
       this.tracks = data.subtitles || [];
       const inUseTracks = this.media ? this.media.textTracks : [];
 
-      this.tracks.forEach((track, index) => {
+      this.tracks.forEach((track: textTrack, index: number) => {
```

Comment:
```suggestion
      this.tracks.forEach((track: any, index: number) => {
```

The tracks don't have a type definition, so we shouldn't add one here until we know what it is.